apps/gitness/src/pages-v2/repo/repo-sidebar.tsx

```diff
@@ -109,6 +116,45 @@ export const RepoSidebar = () => {
     queryParams: { git_ref: normalizeGitRef(fullGitRef || selectedBranchTag?.name) }
   })
 
+  const { data: { body: branches } = {} } = useListBranchesQuery({
```

Comment:
not a blocker, but i'm not convinved about this solution. let's sync with BE team to figure out the expected behaviour.